examples/simple/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Simple Trace Example
+
+This example sets up a tracer in the main application then makes several calls to a custom library instrumented with the <a href="https://github.com/open-telemetry/opentelemetry-cpp"> Open Telemetry SDK </a>. Specifically, it demonstrates the setup of fundamental Open Telemetry componenets such as Tracers, Tracer Providers and Spans.  The example also illustrates context propogation from parent spans to children spans and events.  All telemetry output is directed to stdout.
+
+## Build and Run
+---
+
+<li> Install the latest bazel version by following the steps listed <a href="https://docs.bazel.build/versions/master/install.html">here</a> </li>
+
+<li> Clone or fork the Open Telemetry C++ repository: </li>
+
+```sh
+git clone https://github.com/open-telemetry/opentelemetry-cpp.git
+```
+
+<li> Build the example from the root of the opentelemetry-cpp directory using Bazel: </li>
+
+```sh
+cd opentelemetry-cpp/
+bazel build //examples/simple:example_simple
+```
+
+<li> Run the resulting executable to see telemetry from the application as it calls the instrumented library: </li>
+
+```sh
+bazel-bin/examples/simple/example_simple
+```
+
+## Useful Links
```

Comment:
I think this section should be moved somewhere else. Those links aren't specific to this example but useful to the project as a whole (we wouldn't want to have the same information in all examples).

The same would apply to the _Build and Run_ section. I think it would be better to have more general build instructions at a more central location.

Reply:
I agree the links aren't relevant.  Where would you suggest I move the build/run instructions?  I don't think it would hurt to leave them here as well as duplicate them somewhere else as many interns like myself have never used bazel before

Reply:
I would move those instructions to the CONTRIBUTING doc. [Java](https://github.com/open-telemetry/opentelemetry-java/blob/master/CONTRIBUTING.md#building-opentelemetry-java) has instructions placed there.